pkg/kubernetes/scripts/virtual-machines/utils.jsx

```diff
@@ -46,3 +46,25 @@ export function mouseClick(fun) {
         return fun(event);
     };
 }
+
+/**
+ * Used to get sub-property value.
+ *
+ * Example:
+ *   getValueOrDefault(() => myObj.foo.bar, defaultValue) returns value of myObj.foo.bar if path exists or default otherwise
+ *
+ */
+export function getValueOrDefault(accessor, defaultValue) {
```

Comment:
@bond95, this might be interesting for you as well, in your PR you have this use case.